notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2907 @@
+{
```

Comment:
<p>I don't understand the for loop, so you'r basically taking N items from <code>iter_text</code>  where N equals the amount of non zero values in <code>val_array</code>?</p><p> </p><p>So if I have this input:</p><pre class="ql-syntax" spellcheck="false">val_array = [1,0,0,1]
iter_text = '1234'
</pre><p> </p><p>I'd end up with:</p><pre class="ql-syntax" spellcheck="false">output_arr == ['1', '', '', '2']
</pre><p> </p><p>So you're injecting empty values into those spaces where <code>val_array</code>   is zero, is that observation correct?</p><p> </p><p>If that's the case I'd suggest you to shorten and simplify the func to:</p><pre class="ql-syntax" spellcheck="false">def label_arr(val_array, text_array):
&nbsp;&nbsp;&nbsp;&nbsp;iter_text = iter(text_array)
&nbsp;&nbsp;&nbsp;&nbsp;return list(map(lambda x: next(iter_text) if x else "", val_array))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Yeah,  you understand correctly. If the value is 0 means the test is not run hence labels are not available for that run. 
But we needed labels for all runs for plotting. 
Thanks for the suggestion.

Reply:
Done